frontend/test/__support__/server-mocks/collection.ts

```diff
@@ -10,7 +10,7 @@ import {
 
 export function setupCollectionsEndpoints(
   scope: Scope,
-  collections: Collection[],
+  collections: (typeof ROOT_COLLECTION | Collection)[] = [ROOT_COLLECTION],
```

Comment:
When we don't have any user-created collection we'd only have a root collection and users collection. I omitted the users' collection for simplicity.

This change also enables us to not have to pass any collection when we only want to mock the root collection.

Reply:
This new implicit behavior is not obvious to me. Could you rather call the function like this in the test:
```
setupCollectionsEndpoints(scope, createMockCollection(ROOT_COLLECTiON))
```

One important thing is that `ROOT_COLLECTION` constant is not a collection and it lacks some required properties coming from the API.